README.md

```diff
@@ -56,7 +56,15 @@ By default it will try to communicate with LND using local gRPC connection and `
 You can use `--userest` to start using REST API (default is as mentioned gRPC) - in that case `--rpcserver` can be complete URL to the endpoint.
 
 Of course you can also start the agent on a remote node and just point it to the correct endpoint, but since this is not the default
-use cases flags `--userest` and `--rpcserver` are not advertied in help.
+use cases flags `--userest` and `--rpcserver` are not listed in help.
```

Comment:
create an Advance section in the docs or something like that to describe the unlisted parameters and their usecases

Reply:
Added `Advanced options` section.